### PR TITLE
Adopting cms-export-aws 1.3.0

### PR DESCRIPTION
**Pull request by omidontop**

### CODE REVIEW

File: cms-publish-worker-webapp/pom.xml [hunk 1]
```diff
@@ -12,7 +13,7 @@
 	<parent>
 		<groupId>se.simonsoft</groupId>
 		<artifactId>cms-maven-parent</artifactId>
-		<version>4.1.3</version>
+		<version>8.0.0-SNAPSHOT</version>
```
Comment (takesson) [inline cms-publish-worker-webapp/pom.xml:16]: Finns 8.0.0 nu.

**Comment by takesson** [comment]
add to whitelist

**Comment by omidontop** [comment]
> add to whitelist

Was that to me?

File: cms-publish-worker-webapp/pom.xml (hunk 1 above)
Comment (omidontop) [reply cms-publish-worker-webapp/pom.xml]: Done!

**Comment by takesson** [comment]
Kör `mvn license:format`

**Comment by takesson** [comment]
> > add to whitelist
> 
> Was that to me?

Nope, to Jenkins. :)

**Comment by omidontop** [comment]
> Kör `mvn license:format`

I get a few warnings:

`[INFO] Scanning for projects...
[INFO] 
[INFO] -------< se.simonsoft.publish.worker:cms-publish-worker-webapp >--------
[INFO] Building worker-webapp 0.5.0-SNAPSHOT
[INFO] --------------------------------[ war ]---------------------------------
[INFO] 
[INFO] --- maven-license-plugin:1.10.b1:format (default-cli) @ cms-publish-worker-webapp ---
[INFO] Updating license headers...
[WARNING] Unknown file extension: /media/psf/Home/Projects/simonsoft/cms-publish/cms-publish-worker-webapp/src/test/resources/se/simonsoft/cms/publish/worker/publish-job-has-ticket-completed.json
[WARNING] Unknown file extension: /media/psf/Home/Projects/simonsoft/cms-publish/cms-publish-worker-webapp/src/test/resources/se/simonsoft/cms/publish/worker/publish-job-not-completed.json
[WARNING] Unknown file extension: /media/psf/Home/Projects/simonsoft/cms-publish/cms-publish-worker-webapp/src/test/resources/se/simonsoft/cms/publish/worker/publish-job-options.json
[WARNING] Unknown file extension: /media/psf/Home/Projects/simonsoft/cms-publish/cms-publish-worker-webapp/src/test/resources/se/simonsoft/cms/publish/worker/publish-job-no-ticket.json
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  0.869 s
[INFO] Finished at: 2020-04-09T16:44:25+02:00
[INFO] ------------------------------------------------------------------------
`

**Comment by takesson** [comment]
Jenkins failed the license check once, then passed.

**Comment by takesson** [comment]
The top level aggregate pom does not have a parent POM since it just aggregates the others.

Has worked ok before.

**Comment by takesson** [comment]
> Jenkins failed the license check once, then passed.

Probably some temporary glich while upgrading the parent POM. Let's ignore it unless we see it again.

**Comment by omidontop** [comment]
> The top level aggregate pom does not have a parent POM since it just aggregates the others.
> 
> Has worked ok before.

I see, is it the parent

**Comment by omidontop** [comment]
> > Jenkins failed the license check once, then passed.
> 
> Probably some temporary glich while upgrading the parent POM. Let's ignore it unless we see it again.

Alrighty!

**Pull request merged**